core/src/main/java/org/htsjdk/core/utils/ValidationUtils.java

```diff
@@ -0,0 +1,77 @@
+package org.htsjdk.core.utils;
+
+import java.util.Collection;
+import java.util.function.Supplier;
+
+/**
+ * General utilities to check and validate conditions for constructors/methods.
+ */
+public final class ValidationUtils {

```

Comment:
ParamUtils is marginally shorter which might be good since this is going to show up all over the place.

Reply:
Renamed.